### PR TITLE
Make convex solver more robust for weekday correction

### PR DESCRIPTION
**Pull request by rumackaaron**
### Description
For the doctor visits indicator, we use cvxpy to solve an optimization problem that provides a weekday correction. This solver is not always robust, occasionally it will fail if the magnitude of the objective function is too large. A quick and dirty way to make it more robust is to divide the objective function by some large number.

### Changelog
`weekday.py`: Try up to four times to rescale the objective function, up from two times.

### Fixes 
- No issue linked.

### CODE REVIEW

File: doctor_visits/delphi_doctor_visits/weekday.py [hunk 1]
```diff
@@ -82,14 +82,16 @@ def get_params(data):
...
+            scales = [1, 1e5, 1e10, 1e15]
+            for scale in scales:
+                try:
+                    prob = cp.Problem(cp.Minimize((-ll + lmbda * penalty) / scale))
+                    _ = prob.solve()
+                    break
+                except:
+                    # If the magnitude of the objective function is too large, an error is
+                    # thrown; Rescale the objective function by going through loop
+                    pass
```
Comment (krivard) [inline doctor_visits/delphi_doctor_visits/weekday.py:94]: Consider an `else` block here to throw a useful error if none of the scale values work.

[more info on for/else](https://docs.python.org/3/tutorial/controlflow.html#break-and-continue-statements-and-else-clauses-on-loops)

Comment (rumackaaron) [reply doctor_visits/delphi_doctor_visits/weekday.py:97]: I'm not sure what the ideal behavior is. It's now set up that if the solver fails, the indicator code won't crash, but unless someone is looking through the error logs, this is a pretty silent error.

Comment (krivard) [reply doctor_visits/delphi_doctor_visits/weekday.py:97]: it should either crash or skip outputting the files for the adjusted sensors. At least that way we'll get a sirCAL alert.

Comment (rumackaaron) [reply doctor_visits/delphi_doctor_visits/weekday.py:98]: How does it look now?

**Review by krivard — APPROVED** [review]
Great! @korlaxxalrok can you use your cosmic powers to figure out how we can update the code in Maria's directory on bigchunk-dev-03?

**Pull request merged**